tests/test_host_check.py

```diff
@@ -1144,17 +1216,17 @@ def test_systemd_cpu_mount_error(self, capsys):
                 mock.Mock(returncode=0),
             ],
         )
-        assert output == textwrap.dedent(
+        assert output == pretty_print(
             """\
-            FAIL -- Cgroups
-                    These cgroup mounts do not exist on the host: systemd, cpu.
-                    These mounts are required to run XRd.
-                    If your distro doesn't use systemd, manually add the systemd cgroup mount with:
-                        sudo mkdir /sys/fs/cgroup/systemd
-                        sudo mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
+             FAIL -- Cgroups
+                     These cgroup mounts do not exist on the host: systemd, cpu.
+                     These mounts are required to run XRd.
+                     If your distro doesn't use systemd, manually add the systemd cgroup mount with:
+                         sudo mkdir /sys/fs/cgroup/systemd
+                         sudo mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
             """
         )
-        assert not success
+        assert not result.is_success()
```

Comment:
Would be better to check `result is OverallCheckState.FAILURES` (I don't see any need for the `is_success()` method really).